src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -226,20 +222,13 @@ void GcsActorScheduler::HandleWorkerLeasedReply(
     // node, and then try again on the new node.
     RAY_CHECK(!retry_at_raylet_address.raylet_id().empty());
     actor->UpdateAddress(retry_at_raylet_address);
-    // The backend storage is reliable in the future, so the status must be ok.
-    RAY_CHECK_OK(gcs_actor_table_.Put(
-        actor->GetActorID(), actor->GetActorTableData(), [this, actor](Status status) {
-          RAY_CHECK_OK(status);
-          RAY_CHECK_OK(gcs_pub_sub_->Publish(
-              ACTOR_CHANNEL, actor->GetActorID().Hex(),
-              actor->GetActorTableData().SerializeAsString(), nullptr));
-          Schedule(actor);
-        }));
+    Schedule(actor);
```

Comment:
Don't need to call `Schedule` here, just call `LeaseWorkerFromNode`

Reply:
fixed